Mail-Merge/Phase-by-phase-mail-merge/.NET-Standard/Phase-by-phase-mail-merge/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Phase_by_phase_mail_merge
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStream = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.ReadWrite))
+            {
+                //Open the template document.
+                using (WordDocument document = new WordDocument(fileStream, FormatType.Docx))
+                {
+                    //After executed the mail merge, unexecuted merge fields are removed by default.
+                    //So,set false to maintain the unexecuted merge fields for remaining phase of execution.
+                    document.MailMerge.ClearFields = false;
```

Comment:
//Disable the flag to maintain unmerged fields in document for next phase of execution